charts/pb/values.yaml

```diff
@@ -81,6 +81,7 @@ featureFlags:
   useCapabilitiesSelector: "true"
   showCovid19: "true"
   showOrderForm: "true"
+  showDfocvc: "false"
```

Comment:
Hi Keith, I think we want this to default to true (and then be turned off at the next level up until it's ready to go) please.
Thanks
Gavin

Reply:
(sorry, next level up is the master chart in platform-helm)